.gitlab/ci/.gitlab-ci.global.yml

```diff
@@ -438,69 +440,7 @@
         ./Tests/scripts/sdk_pylint_check.sh
       fi
     - section_end "Test Infrastructure"
-    - rm -rf Packs/MyPack # remove MyPack from the Packs folder, workaround until demisto-sdk 1.27.5 will be released
```

Comment:
at line 400 rename `.unittests-and-lint-settings` to `.pre-commit-settings`

Reply:
renamed